src/site/stages/build/process-cms-exports/transformers/paragraph-situation_update.js

```diff
@@ -0,0 +1,35 @@
+const { getDrupalValue } = require('./helpers');
+const moment = require('moment-timezone');
+
+const transform = entity => {
+  const {
+    fieldDateAndTime,
+    fieldSendEmailToSubscribers,
+    fieldWysiwyg,
+  } = entity;
+  return {
+    contentModelType: entity.contentModelType,
+    entity: {
+      entityType: 'paragraph',
+      entityBundle: 'situation_update',
+			fieldDateAndTime: {
+				// Assume the raw data is in UTC
+				date: moment
+				.tz(getDrupalValue(fieldDateAndTime), 'UTC')
+				.format('YYYY-MM-DD HH:mm:ss UTC'),
```

Comment:
I normally wouldn't question this, but I did just run into [this](https://github.com/department-of-veterans-affairs/vets-website/pull/13456/files#diff-a4e88f3d786dff175a5cd7b02d9343d8R12-R18) earlier today.